docs/components/movement-sensor/gps/dual-gps-rtk.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Configure a Dual GPS Movement Sensor"
+linkTitle: "dual-gps-rtk"
+weight: 10
+type: "docs"
+description: "Configure a movement sensor that calculates compass heading from two gps movement sensors."
+images: ["/icons/components/imu.svg"]
+# SMEs: Rand
+---
+
+The `dual-gps-rtk` model of movement sensor calculates compass heading from two gps movement sensors.
+In addition to [`CompassHeading()`](/components/movement-sensor/#getcompassheading), this model provides data for [`GetPosition()`](/components/movement-sensor/#getposition).
```

Comment:
```suggestion
In addition to [`CompassHeading()`](/components/movement-sensor/#getcompassheading), this model provides data for [`Position()`](/components/movement-sensor/#getposition).
```

Either use `Get` as a prepend for both or don't. The `go` APIs follow the go style of not prefixing `Get` to the API function names, the other SDKs do.

Reply:
Sounds good-- will use Get to match docs I think